registry/pressSettings.xml

```diff
@@ -131,8 +131,8 @@
 		<name>cataloguingMetadata</name>
 		<value>
 			<array>
-				<element key="name">{translate key="manager.setup.category"}</element>
```

Comment:
It looks like you might be able to get rid of `manager.setup.category` entries in the locale files now...